synapse/lib/cell.py

```diff
@@ -1012,6 +1012,7 @@ async def __anit__(self, dirn, conf=None, readonly=False, parent=None):
         self.isactive = False
         self.inaugural = False
         self.activecoros = {}
+        self._checkspace = None
```

Comment:
Is there a reason to not initialize this as an asyncio.Eeent directly?

Reply:
Not really, we might not need it but one extra event probably isn't a big deal

Reply:
asyncio events are very cheap ( unlike threading events ) so we can make them without issue.